tarantool/connection.py

```diff
@@ -766,14 +768,20 @@ def _read_response(self):
         # Read the packet
         return self._recv(length)
 
-    def _send_request_wo_reconnect(self, request):
+    def _send_request_wo_reconnect(self, request, on_push=None, on_push_ctx=None):
         """
         Send request without trying to reconnect.
         Reload schema, if required.
 
         :param request: Request to send.
         :type request: :class:`~tarantool.request.Request`
 
+        :param on_push: Сallback for processing out-of-band messages.
+        :type on_push: :obj:`function`, optional
+
+        :param on_push_ctx: Сontext for working with on_push callback.
```

Comment:
Btw, you can use 
```
:paramref:`~tarantool.Connection._send_request_wo_reconnect.params.on_push`
```
here, but it's up to you.

Reply:
Thanks! Left unchanged